applications/packages/auth/common.auth-sdk.client/src/sdk/state.ts

```diff
@@ -0,0 +1,18 @@
+import { Store } from "@wraithlight/core.redux";
+
+import { GlobalState } from "./state.model";
```

Comment:
This should be something like AuthState what contains SessionState and AccounState